### ISSUE TITLE
Closing the flowgraph hangs Gnuradio

### ISSUE THREAD

**Issue by gasparka** [myriadrf/gr-limesdr#1]
Closing from the X button.
Following message loops for about 10 sec:
```bash
##################
libusb: warning [add_to_flying_list] failed to arm first timerfd (errno 9)
Write(64 bytes) failed
libusb: warning [add_to_flying_list] failed to arm first timerfd (errno 9)
Write(64 bytes) failed
libusb: warning [add_to_flying_list] failed to arm first timerfd (errno 9)
Write(64 bytes) failed
ERROR: Write(64 bytes) failed

##################
libusb: warning [add_to_flying_list] failed to arm first timerfd (errno 9)
Write(64 bytes) failed
```

**Comment by yakorev** [comment]
Hi, gasparka!
I have the same problem, but i noticed that selecting WXGui in the flowgraph doesnot get error when i closing it. The output is like in tutorial "Disconnected from device number 0" 😃

**Comment by gasparka** [comment]
I found that 'Kill Graph' works when using QT GUI.

**Comment by yakorev** [comment]
'Kill Graph' for me works too, but it does not perform stream close and device reset routines, as said in https://wiki.myriadrf.org/Gr-limesdr_Plugin_for_GNURadio

**Comment by ztamosevicius** [comment]
Fixed in commit a283ec413b029eedee3f19952286bdcbbc79c34f
